library/src/main/java/com/novoda/downloadmanager/lib/Constants.java

```diff
@@ -108,6 +108,11 @@
     public static final int TAR_BLOCK_SIZE = 512;
 
     /**
+     * The size of a tar record
+     */
+    public static final int TAR_RECORD_SIZE = TAR_BLOCK_SIZE * 20;
```

Comment:
>The new approach is more resilient as it will read by records of 20 blocking factor (default blocking factor of tar)

You mentioned parameterising the 20 - yday I recommended doing it as part of this PR and leave 20 as the default. but now thinking on it, yagni; let's do it when someone raises the issue. The 20 can be extracted as a private constant though as `DEFAULT_TAR_BLOCKING_FACTOR` as it's not an arbitrary number.